packages/cli/command_spec.pony

```diff
@@ -325,22 +343,66 @@ class val ArgSpec
     _name + "[" + _typ.string() + "]" +
       if not _required then "(=" + _default.string() + ")" else "" end
 
-type _Value is (Bool | String | I64 | F64)
+class _StringSeq is ReadSeq[String]
```

Comment:
Let me give that a look. I started with this simple wrapper class to get a Stringable and a String sequence of some kind. But a persistent structure might make more sense given the way it has to be updated during parse.